test/LinkTest/Main.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2010-2020, Tarantool AUTHORS, please see AUTHORS file.
```

Comment:
*`Copyright 2010-2024`

Reply:
Fixed. Also, added missing copyright to the header.